function-reference/show-alert.md

```diff
@@ -0,0 +1,21 @@
+---
+description: >-
+  Show a temporary notification style alert message to the user
+---
+
+# showAlert
+
+## Signature
+
+```text
+showAlert(message, style) -> void
+    
+```
+
+#### Arguments
+
+| Argument Name | Description |
+| message | string |
+| style | string |
+
+![Click to expand](../.gitbook/assets/showAlert.gif)
```

Comment:
This gif is not correct. Instead of showing a system error alert, please create a new app, drag and drop a button, on the `onClick` property of this button, set this to showAlert and then click the button.